documentation/asciidoc/computers/os/updating.adoc

```diff
@@ -120,13 +120,13 @@ sudo apt purge tree
 
 WARNING: Pre-release versions of software are not guaranteed to work. You should not use `rpi-update` on any system unless recommended to do so by a Raspberry Pi engineer. It may leave your system unreliable or even completely broken. It should not be used as part of any regular update process.
 
-The `rpi-update` script is supplied by a third party, "Hexxeh", and also supported by Raspberry Pi engineers. The script source can be found on GitHub at https://github.com/Hexxeh/rpi-update
+The `rpi-update` script (originally written by https://github.com/Hexxeh[Hexxeh]) is supported by Raspberry Pi engineers. The script source is on https://github.com/raspberrypi/rpi-update[GitHub]
 
 ==== What it does
 
 `rpi-update` will download the latest pre-release version of the linux kernel, its matching modules, device tree files, along with the latest versions of the VideoCore firmware. It will then install these files to relevant locations on the SD card, overwriting any previous versions.
 
-All the source data used by `rpi-update` comes from the GitHub repo https://github.com/Hexxeh/rpi-firmware. This repository simply  contains a subset of the data from the https://github.com/raspberrypi/firmware[official firmware repository], as not all the data from that repo is required.
+All the source data used by `rpi-update` comes from the https://github.com/raspberrypi/rpi-update[GitHub repo]. This repository simply  contains a subset of the data from the https://github.com/raspberrypi/firmware[official firmware repository], as not all the data from that repo is required.
```

Comment:
According to the PR you linked to, `https://github.com/Hexxeh/rpi-firmware` has now changed to `https://github.com/raspberrypi/rpi-firmware` ?

Reply:
Good catch, I didn't notice there was a second repo 🙂